src/XMakeBuildEngine/UnitTestsPublicOM/Definition/Project_Tests.cs

```diff
@@ -2917,7 +3017,7 @@ public void GetAllGlobsShouldFindIndirectlyReferencedGlobsFromProperties()
 
             var expected = new GlobResultList
             {
-                Tuple.Create("A", "*", new HashSet<string> {"*"}),
+                Tuple.Create("A", "*", new HashSet<string> {"*"}.ToImmutableHashSet()),
```

Comment:
Should be able to go from array to immutable hash set directly without the mutable `HashSet` in between, right?